Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -64,6 +67,15 @@ ChromeDriver CreateDriverInner(ChromeDriverService service)
 
             options.SetCommonChromiumOptions(configuration);
 
+            // While the Edge driver easily locates Edge in Windows, it struggles in Linux, where the different release
+            // channels have different executable names. This setting looks up the "microsoft-edge-stable" command and
+            // sets the full path as the browser's binary location.
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) &&
+                CliWrapHelper.WhichAsync("microsoft-edge-stable").GetAwaiter().GetResult().FirstOrDefault() is { } binaryLocation)
```

Comment:
Use async-await instead. I know it'll be a pain to change everything upwards, but the ultimate caller in `UITestExecutionSession` is async, so it'll be a mechanical change.